extras/docs/troubleshooting.rst

```diff
@@ -0,0 +1,86 @@
+Troubleshooting Guide
+=====================
+
+First check : mod installation and configuration
+------------------------------------------------
+
+Go to your *KSP/GameData* folder. You should see a *KerbalSimpit* folder if SimPit is installed (else, install it with CKAN).
+
+Open the configuration file *KSP/GameData/KerbalSimpit/PluginData/Settings.cfg*
+
+The line ``PortName = COM3`` should be updated with the correct port number. It should be identical to the one used with the Arduino IDE.
```

Comment:
On a linux or Mac the USB port ID will look like /dev/cu.usbmodem401 OR /dev/ttyUSBO

Reply:
Fixed on the next commit. Thanks